etna/analysis/plotters.py

```diff
@@ -682,30 +690,56 @@ def plot_correlation_matrix(
         Segments to use
     method:
         Method of correlation:
-
         * pearson: standard correlation coefficient
-
         * kendall: Kendall Tau correlation coefficient
-
         * spearman: Spearman rank correlation
-
+    columns:
+        Columns to use, if None use all columns
     figsize:
         size of the figure in inches
     """
     if segments is None:
         segments = sorted(ts.segments)
+
     if "vmin" not in heatmap_kwargs:
         heatmap_kwargs["vmin"] = -1
     if "vmax" not in heatmap_kwargs:
         heatmap_kwargs["vmax"] = 1
 
-    correlation_matrix = get_correlation_matrix(ts, segments, method)
-    fig, ax = plt.subplots(figsize=figsize)
-    ax = sns.heatmap(correlation_matrix, annot=True, fmt=".1g", square=True, ax=ax, **heatmap_kwargs)
-    labels = list(ts[:, segments, :].columns.values)
-    ax.set_xticklabels(labels, rotation=45, horizontalalignment="right")
-    ax.set_yticklabels(labels, rotation=0, horizontalalignment="right")
-    ax.set_title("Correlation Heatmap")
+    if mode not in ["macro", "per-segment"]:
+        raise ValueError(f"'{mode}' is not a valid method of mode.")
+
+    if mode == "macro":
+        fig, ax = plt.subplots(figsize=figsize)
+        correlation_matrix = get_correlation_matrix(ts, columns, segments, method)
+
+        if columns:
```

Comment:
Here it is better to compare this way `columns is not None`

Reply:
Set the value of the `columns` explicitly in the begging of the method to avoid so much ifs